### PR TITLE
Apply ufmt to torch internal

### PR DESCRIPTION
**Pull request by huydhn**
This is a big bang PR, merge conflicts are probably expected and will be addressed at merge.

### CODE REVIEW

File: torch/_tensor.py [hunk 1]
```diff
@@ -53,7 +65,10 @@ def _rebuild_from_type_v2(func, new_type, args, state):
     # Tensor does define __setstate__ even though it doesn't define
     # __getstate__. So only use __setstate__ if it is NOT the one defined
     # on Tensor
-    if getattr(ret.__class__, "__setstate__", Tensor.__setstate__) is not Tensor.__setstate__:
+    if (
+        getattr(ret.__class__, "__setstate__", Tensor.__setstate__)
+        is not Tensor.__setstate__
+    ):
```
Comment (kit1980) [inline torch/_tensor.py:71]: This seems much worse than before for me.

File: torch/_python_dispatcher.py [hunk 2]
```diff
@@ -48,14 +49,20 @@
...
-        "XLA", "AutogradXLA",
-        "Lazy", "AutogradLazy",
+        "CPU",
+        "AutogradCPU",
+        "FPGA",
+        "AutogradOther",
+        "XLA",
+        "AutogradXLA",
+        "Lazy",
+        "AutogradLazy",
```
Comment (kit1980) [inline torch/_python_dispatcher.py:65]: Someone carefully aligned the keys to convey relationship, and after autoformat it's gone.

**Review by kit1980 — COMMENTED** [review]
This formatter seems to be very opinionated. I highlighted just couple of places where I think it decreased readability. I don't think all of that is required by PEP 8.
Maybe there are settings we can tweak to make it less aggressive?

**Comment by kit1980** [comment]
@ezyang do you think we should do this?

File: torch/_python_dispatcher.py (hunk 2 above)
Comment (huydhn) [reply torch/_python_dispatcher.py:65]: You're right in this case. The original format is easier to read. I think we can make an exception for this one (and any others that is standing out like this)

File: torch/_tensor.py (hunk 1 above)
Comment (huydhn) [reply torch/_tensor.py:71]: Well, AFAIK this is the same kind of setup used internally by fbsource. So I opt for this by default for consistency. Let me see if we can tweak this

**Comment by huydhn** [comment]
> @ezyang do you think we should do this?

+1 To be honest, I'm jumping the gun a bit here.  At the moment, only the following files are formatted like this (by BLACK)

```
'torch/package/**/*.py',
'torch/onnx/**/*.py',
'torch/_refs/**/*.py',
'torch/_prims/**/*.py',
'torch/_meta_registrations.py',
'torch/_decomp/**/*.py',
```

If we consider that this is too big a change in the core codebase, I will rework this to cover only the same set of files as BLACK so that we can remove BLACK. For the context, the new UFMT linter is BLACK + USORT so having BLACK becomes redundant.

**Comment by ezyang** [comment]
I reviewed the pessimized sites and I don't think it is worse enough to warrant not doing this. In other words, let's ship as is

**Review by ezyang — APPROVED** [review]

File: torch/_python_dispatcher.py (hunk 2 above)
Comment (huydhn) [reply torch/_python_dispatcher.py]: For this specific case, I will add `# fmt: off` and `# fmt: on` to keep it as before. That's also the general way to tell BLACK to not format certain part of the code base.

**Comment by IvanYashchuk** [comment]
Is there a test that ensures the formatting will stay consistent or these changes would happen once in a while? Is ufmt included in lintrunner?

**Comment by huydhn** [comment]
> Is there a test that ensures the formatting will stay consistent or these changes would happen once in a while?

If I interpret your question correctly, the test you are thinking about is the lintrunner check, which runs for every PR. So if someone make change to a file that has been formatted and introduce some inconsistency syntax, lintrunner check will stop it. Of course, the change owner can fix it by running `lintrunner -a` for the new change as usual.

> Is ufmt included in lintrunner?

Yes, it has been added to [lintrunner](https://github.com/pytorch/pytorch/blob/master/.lintrunner.toml#L685-L701), so you can use lintrunner as usual. Note that not everything is covered yet. I'm gradually adding more and more files there like this PR.

**Pull request closed without merging**